Apps/APN/src/services/CustomerIOService.js

```diff
@@ -1,31 +1,19 @@
-import {
-  CioLogLevel,
-  CustomerIO,
-  CustomerIOEnv,
-  CustomerioConfig,
-  InAppMessageEventType,
-} from 'customerio-reactnative';
-
+import { CustomerIO, CioConfig } from 'customerio-reactnative';
 export const initializeCustomerIoSDK = (sdkConfig) => {
-  const env = new CustomerIOEnv();
-  env.siteId = sdkConfig.siteId;
-  env.apiKey = sdkConfig.apiKey;
-
-  const config = new CustomerioConfig();
-  config.enableInApp = true;
-
-  if (sdkConfig.debugMode) {
-    config.logLevel = CioLogLevel.debug;
-  }
-  if (sdkConfig.trackingUrl) {
-    config.trackingApiUrl = sdkConfig.trackingUrl;
-  }
-  // Advanced SDK configurations only required by sample app, may not be required by most customer apps
-  config.autoTrackDeviceAttributes = sdkConfig.trackDeviceAttributes;
-  config.backgroundQueueMinNumberOfTasks = sdkConfig.bqMinNumberOfTasks;
-  config.backgroundQueueSecondsDelay = sdkConfig.bqSecondsDelay;
-
-  CustomerIO.initialize(env, config);
+  const config = {
+    cdpApiKey: 'cdp_api_key', // Mandatory
+    migrationSiteId: 'site_id', // For migration
+    trackApplicationLifecycleEvents: true, // TODO: Update this to a configurable property based on settings
+    flushAt: sdkConfig.bqMinNumberOfTasks,
+    flushInterval: sdkConfig.bqSecondsDelay,
+    inApp: {
+	    siteId: 'site_id',
+    }
+ };
+ if (sdkConfig.debugMode) {
+  config.logLevel = CioLogLevel.debug;
+}
+CustomerIO.initialize(config)
```

Comment:
More config options will be added later as we progress on the milestones and tickets.

Reply:
Can we add update `CustomerIoSDKConfig` as well? as in if we just add the new attributes without removing the old ones, we would be able to utilize the new attributes in here and the app wont stop compiling either. This was we can test these PR as well and not wait till we have made all changes

Reply:
The name for configuration object has been updated from `CustomerioConfig` to  `CioConfig`. It might not be possible to test until we update the native module to use CDP SDK as the new properties can only be tested with Datapipelines.